synapse/common.py

```diff
@@ -1368,3 +1368,12 @@ def _timeout(delay):
     """
     loop = asyncio.get_running_loop()
     return _Timeout(loop.time() + delay if delay is not None else None)
+
+def format(text, **kwargs):
```

Comment:
Can we memoize this?

Reply:
I guess we could, but i doubt its worth it for the current use case?

Reply:
Maybe. How often are those strings interpolated? Probably every time someone loads the data model explorer. Maybe with `syn:form`/`syn:prop`s too?

Reply:
nope. one-time load events :+1: